src/template.c

```diff
@@ -50,7 +50,7 @@ char *templateRender(Template *template)
     rewind(file);
 
     buff = malloc(sizeof(char) * (len + 2));
-    fread(buff + 1, sizeof(char), len, file);
+    (void)!fread(buff + 1, sizeof(char), len, file);
```

Comment:
I dislike the way to introduce a logical NOT operator here. Can you use alternative ones?

Reply:
Any progress so far?

Reply:
> Any progress so far?

Update GCC or change the compiler would dismiss this warning message,
(I got that warning at GCC 9.1, but it disappears at GCC 12.1 now)